src/main/groovy/org/mini2Dx/butler/task/PushTask.groovy

```diff
@@ -49,11 +49,11 @@ class PushTask extends DefaultTask  {
 	}
 	
 	def pushAnyOsBuild() {
-		if(project.getExtensions().findByName('butler').anyOs == null) {
+		if(project.getExtensions().findByName('butler').anyOs.binDirectory == null) {
```

Comment:
But anyOs is never null, even if you don't define it in the build file it still has a value and so butler always pushes it.